wrappers/java/src/test/java/org/hyperledger/indy/sdk/did/GetEndpointForDidTest.java

```diff
@@ -32,6 +32,8 @@ public void testGetEndpointForDidWorksFromLedger() throws Exception {
 		String attribRequest = Ledger.buildAttribRequest(trusteeDid, trusteeDid, null, endpoint, null).get();
 		Ledger.signAndSubmitRequest(pool, wallet, trusteeDid, attribRequest).get();
 
+		Thread.sleep(5_000);
```

Comment:
Do we need switching to ensurePreviousRequestApplied ?

Reply:
Only via one more GET_ATTRIB request to ledger and still without guaranties